packages/loot-core/src/server/main.ts

```diff
@@ -1482,16 +1482,18 @@ handlers['subscribe-change-password'] = async function ({ password }) {
 };
 
 handlers['subscribe-sign-in'] = async function ({ password }) {
-  const res = await post(getServer().SIGNUP_SERVER + '/login', {
-    password,
-  });
+  try {
+    const res = await post(getServer().SIGNUP_SERVER + '/login', {
+      password,
+    });
 
-  if (res.token) {
-    await asyncStorage.setItem('user-token', res.token);
-    return {};
+    if (res.token) {
+      await asyncStorage.setItem('user-token', res.token);
+      return {};
+    }
+  } catch (e) {
+    return { error: 'invalid-password' };
   }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we are missing a return statement if `res.token` is not set or empty. While _theoretically_ it should never happen - we should still account the off chance that it does happen. Maybe throwing a new error would be sufficient in this case.

Reply:
Good idea, added now. Looking at the rest of the file I'm not sure what the convention with thrown error messages is so if you'd prefer it to read differently please let me know!